src/tests/IntegrationTests/Events/RepositoryManagerTests.cs

```diff
@@ -162,6 +174,7 @@ await RepositoryManager
 
                 repositoryManagerListener.Received().OnIsBusyChanged(Args.Bool);
                 repositoryManagerListener.DidNotReceive().CurrentBranchUpdated(Args.NullableConfigBranch, Args.NullableConfigRemote);
+                repositoryManagerListener.DidNotReceive().GitAheadBehindStatusUpdated(Args.GitAheadBehindStatus);
```

Comment:
Commit operations will affect the ahead/behind status. We should update this here.